src/meta/meta_ckpt.c

```diff
@@ -1483,10 +1483,11 @@ __wt_meta_ckptlist_set(
                 WT_ERR(__ckpt_check_backup_blocks(session, ckpt, fname));
         }
 
+    /* If a checkpoint LSN is provided this is where recovery will resume from. */
```

Comment:
Minor nit: Ending the sentence with 'from' is awkward. I suggest something like:
"If given, the metadata needs the LSN of this checkpoint for recovery."

Reply:
SGTM. Updated in https://github.com/wiredtiger/wiredtiger/pull/11056/commits/351f7ee7b8b6fc497fc2411c6bf1bf4ef0de5823